lib/resources/example_json/soarian_r4_examples_procedure.rb

```diff
@@ -0,0 +1,344 @@
+              # rubocop:enable Layout/LineLength
+            }
+          }
+        }
+      ],
+      "link": [
+        {
+          "relation": 'self',
+          "url": 'https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Procedure?patient=A879904FD2FE4B2D90C89FDA84E1285F'
+        }
+      ]
+    }.freeze
+
+    SOARIAN_R4_PROCEDURE_ERROR ||= {
+      "resourceType": 'Procedure',
+      "id": '7DD361F7EB43448690AABCCC45E62285.4010',
+      "meta": {
+        "lastUpdated": '2020-07-16T20:32:34-04:00'
+      },
+      "status": 'entered-in-error',
+      "code": {
+        "coding": [
+          {
+            "system": 'http://snomed.info/sct',
+            "code": '429211003',
+            "display": 'Procedure for Atrial Fib'
+          }
+        ],
+        "text": 'Procedure for Atrial Fib'
+      },
+      "subject": {
+        "reference": 'Patient/7DD361F7EB43448690AABCCC45E62285',
+        "display": 'Lansford,Mukesh-Procedure Dianne'
+      },
+      "encounter": {
+        "reference": 'Encounter/7DD361F7EB43448690AABCCC45E62285.47027',
+        "display": 'City General,Initial'
+      },
+      "performedDateTime": '2020-07-16T02:55:00-04:00',
+      "focalDevice": [
+        {
+          "action": {
+            "coding": [
+              {
+                "system": 'http://snomed.info/sct',
+                "code": '386746003',
+                "display": 'Maintenance - action'
+              }
+            ]
+          },
+          "manipulated": {
+            "reference": 'Device/7DD361F7EB43448690AABCCC45E62285.3007'
+          }
+        }
+      ],
+      "text": {
+        "status": 'generated',
+        # rubocop:disable Layout/LineLength
+        "div": '<div xmlns="http://www.w3.org/1999/xhtml"><b>Procedure</b><br /><b>Patient</b>:Lansford,Mukesh-Procedure Dianne;<br /><b>Encounter</b>:City General,Initial;<br /><b>Status</b>:entered-in-error;<br /><b>Code</b>:<br /><b>system</b>:http://snomed.info/sct <b>code</b>:429211003 <b>display</b>:Procedure for Atrial Fib;<br /><b>Performed</b>:2020-07-16T02:55:00-04:00;<br /><b>Focal Device</b>: <br /><b>Action</b>: <b>system</b>:http://snomed.info/sct <b>code</b>:386746003 <b>display</b>:Maintenance - action;<br /><b>Manipulated</b>:Device/7DD361F7EB43448690AABCCC45E62285.3007;<br /></div>'
+        # rubocop:enable Layout/LineLength
+      }
+    }.freeze
+
+    SOARIAN_R4_PROCEDURE_STOPPED ||= {
```

Comment:
I do not see any request URL is using this response in procedure.md file. Can we add it?